docs/docs/security.md

```diff
@@ -0,0 +1,50 @@
+- [Glossary](/docs/glossary.md) - Keep this page handy for when you run into new or unfamiliar terminology.
+
+## TLS Certificates
+
+The long-time standard for server identity verification, the use of TLS certificates has exploded since [Let's Encrypt](https://letsencrypt.org/) made it possible for anyone to get a trusted certificate for free.
+
+- The [Certificates](/docs/topics/certificates.md) topic page covers several basic methods for generating trusted or testing certificates.
+- Our article on [Installing Pomerium Using Helm](/docs/k8s/helm.md) touches [briefly](/docs/k8s/helm.md#install-and-configure-cert-manager) on using [cert-manager](https://cert-manager.io/docs/) to manage certificates in Kubernetes environments. We also wrote a guide for their docs site covering integration of the [Pomerium Ingress](https://cert-manager.io/docs/tutorials/acme/pomerium-ingress/) Controller with cert-manager.
+- The [Upstream mTLS With Pomerium](/guides/upstream-mtls.md) guide demonstrates mTLS between Pomerium and upstream services.
+- Depending on your environment's needs, you may choose to verify some of all of your end users with [Client-Side mTLS](/guides/mtls.md).
+
+## User Identity and Context
+
+Part of Pomerium's strength comes from the ability to pass user identity and context to your upstream service. This enables repeated verification of authorization throughout a system.
+
+- [Getting the user's identity](/docs/topics/getting-users-identity.md) details the JWT Pomerium creates to identify the user in any given request.
+- [Original User Context](/docs/topics/original-request-context.md) explains how to pass along the user context when upstream services communicate with each other to complete a request.
+- Many applications support native JWT verification. See [Enable jWT Authentication in Grafana](/guides/grafana.md#enable-jwt-authentication-in-grafana) for an example. For those that don't, you can perform [JWT Verification](/guides/jwt-verification.md) with a sidecar.
+
+
+## Device Identity
+
+Often overlooked or confused with multi-factor-authentication, device identity (and posture) is one of the most important and under-utilized aspects of a strong zero trust security model.
+
+- [Device Identity](/docs/topics/device-identity.md) provides background on the concept, and points the reader on how to configure policies that use device identity, and enroll devices in both open-source and Enterprise environments.
+
+## Service Mesh
+
+If you've read through all the docs linked above, first of all *wow*. That's a lot to absorb, kudos to you. But if you got this far and you're overwhelmed thinking about how to manage mutual authentication, user context verification, etc, between all your various applications, then you're primed and ready for a **service mesh**. A service mesh is a software component that acts as an infrastructure layer to facilitate the communication (and authentication) between services.
```

Comment:
```suggestion
After you have read through all the docs linked above and absorbed them, you will start thinking about managing mutual authentication, user context verification, and more between your various applications. The next step is to apply this knowledge to a **service mesh**. A service mesh is a software component that acts as an infrastructure layer to facilitate the communication (and authentication) between services.
```

While I enjoyed the original, I'm worried some might take it the wrong way, so here's a swift edit with hopefully the same result.

Reply:
I'm open to adjusting this, but replacing `If` with `After` implies that the user must/should read everything linked, which is not something I want to infer.

I'm conflicted on removing the contraction `you've`, as I like writing such that it reads as if spoken.

I'm gonna leave this as is until @desimone takes a look and can weigh in on the informal tone set by the original.

Reply:
I'm comfortable with the informal tone. It's okay to have a bit of fun.